test/frontend/components/GitHubRepoUrlField.test.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { expect } from 'chai';
+
+import GitHubRepoUrlField, { required, githubRepoUrl } from '../../../frontend/components/GitHubRepoUrlField';
+import RenderField from '../../../frontend/components/GitHubRepoUrlField/RenderField';
+
+describe('<GitHubRepoUrlField />', () => {
+  it('renders', () => {
+    const props = {
+      id: 'boopId',
+      name: 'boopName',
+    };
+
+    const wrapper = shallow(<GitHubRepoUrlField {...props} />);
+    expect(wrapper).to.be.defined;
+    expect(wrapper.find('Field[id="boopId"]')).to.have.length(1);
+  });
+
+  it('has validators specified', () => {
+    const props = {
+      id: 'boopId',
+      name: 'boopName',
+    };
+
+    const wrapper = shallow(<GitHubRepoUrlField {...props} />);
+    const validateProp = wrapper.props().validate;
+    expect(validateProp.length).to.equal(2);
```

Comment:
I also don't love this way of testing that the validation functions are "hooked" up to the component. Couldn't think of anything better unfortunately.